.github/workflow/ci.yml

```diff
@@ -0,0 +1,160 @@
+name: continuous-integration
```

Comment:
As I said via DM, I think best to remove this file for now, and instead merge  a PR (which I'll make tomorrow) that contains our working GHA, and then tweak that to do the full build.

Reply:
OK !